### ISSUE TITLE
Handling cell deletion

### ISSUE THREAD

**Issue by recaeadmin** [j-fu/ExtendableGrids.jl#65]
Hi @j-fu,

Great work with this meshing package! Particularly appreciate the partitioning feature! 🚀 

Quick question - have you ever considered the problem of cell deletion? Let's just imagine a heat conduction problem and a melting material. If one cell reached the melting temperature, we ought to remove it because it "melted" away.

I have not found documentation or discussion about this in ExtendableGrids, would you happen to have any idea?

**Comment by j-fu** [comment]
No, we don't have this and didn't think about this so far. It seems to be possible to implement something like this based on a marker vector.  Essentially this would amount to creating a new grid because removing cells also would lead to removing vertices.  One would have to figure out what to do with the boundary. However, something like this is  already in the subgrid code. May be tweaking cellregions and extracting subgrids could work.
